app/buildpacks/java_buildpack.rb

```diff
@@ -17,6 +20,10 @@ def base_images_registry_url
     Configurations.docker.base_images_registry_url
   end
 
+  def base64_maven_setting
+    Configurations.java.base64_maven_setting
```

Comment:
This needs to be plural to match the addition in the `configurations.rb` file. Might as well, then, make the method name plural so it all matches as well, which would also require a final change on L#8.

Reply:
Nice catch mate! Thanks a lot! =)